src/main/java/io/github/rabobank/shadow_tool/ShadowFlow.java

```diff
@@ -33,6 +33,10 @@
     private static final String INSTANCE_PREFIX_FORMAT = "[instance=%s]";
     private static final String DEFAULT_INSTANCE_NAME = "default";
     private static final String MESSAGE_FORMAT = "{} Calling new flow: {}";
+    private static final String DEFAULT_ALGORITHM = "RSA";
+    private static final String DEFAULTALGORITHM_MODE_PADDING =
```

Comment:
Is there no "_" on purpose between default and algorithm?

Reply:
If a mistake was the purpose, then yes